File: Gol-Domain/Entities/Airplane.cs
using System.ComponentModel.DataAnnotations.Schema;

namespace Gol_Domain.Entities
{
    public class Airplane : BaseEntity
    {
        public string Code { get; set; }
        public int NumberOfPassengers { get; set; }
        public int AirplaneModelId { get; set; }

        [ForeignKey("AirplaneModelId")]
        public AirplaneModel AirplaneModel { get; set; }
    }
}

File: FrontEnd/GolAirlines/src/Models/Airplane.ts
import { AirplaneModel } from "./AirplaneModel";

export class Airplane {
  id: number;
  code: string;
  airplaneModelId: number;
  model: AirplaneModel;
  numberOfPassenger: number;
  createdAt: Date;
  updatedAt: Date;
}
